py4syn/epics/DxpClass.py

```diff
@@ -17,14 +17,17 @@
 from threading import Event
 from py4syn.utils.timer import Timer
 import os
+import h5py
 
 
 class Dxp(StandardDevice, ICountable):
     # CONSTRUCTOR OF DXP CLASS
-    def __init__(self, mnemonic, output, numberOfChannels=4, numberOfRois=32,
-                 pv=None, dxpType="mca", responseTimeout=15):
+    def __init__(self, mnemonic, numberOfChannels=4, numberOfRois=32,
+                 pv=None, dxpType="mca", responseTimeout=15, output="./out",
+                 imageDeep=2048):
```

Comment:
What is this ```imageDeep```?
Is this defined once in the device and remain the same for the whole use of it?
The name is a bit confusing.